tests/conditional_compilation/tools/infer_tool.py

```diff
@@ -0,0 +1,69 @@
+# !/usr/bin/env python3
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint:disable=invalid-name,no-name-in-module,logging-format-interpolation,redefined-outer-name
+
+""" Tool for running inference and storing results in npz files.
+"""
+import argparse
+import logging as log
+import sys
+import os
+import numpy as np
+from openvino.inference_engine import IENetwork, IECore
+
+log.basicConfig(format="[ %(levelname)s ] %(message)s", level=log.INFO, stream=sys.stdout)
+
+
+def infer(ir_path, device):
+    """
+     Function to perform IE inference using python API "in place"
+    :param ir_path: Path to XML file of IR
+    :param device: Device name for inference
+    :return: Dict containing out blob name and out data
+    """
+
+    bin_path = os.path.splitext(ir_path)[0] + '.bin'
+    net = IENetwork(model=ir_path, weights=bin_path)
+    feed_dict = {}
+    for layer, name in net.inputs.items():
+        feed_dict.update({layer: np.ones(shape=name.shape)})
```

Comment:
please, create a separate function with input preparation. Function docstring should describe the aim that input should be reproducible from run to run